test/test_helper.rb

```diff
@@ -5,7 +5,14 @@
   add_group 'DSL', 'lib/convection/dsl'
 end
 
+
 gem 'minitest'
 require 'minitest/autorun'
 require 'minitest/spec'
 require_relative '../lib/convection'
+
+module Minitest::Assertions
+  def assert_nothing_raised(*)
```

Comment:
This method literally only calls the block provided (ignoring all arguments). You should define this method so that it raises an assertion error if anything *was* raised.